### DISCUSSION TITLE
GHCR images having dot in the name are not accessible in UI

### DISCUSSION THREAD

**Discussion by plutzilla** [community/community#25628] [Actions]
<p>Docker images having dot in the repository name are not accessible via Packages UI when pushed to Github Container Registry (<a href="http://ghcr.io" rel="nofollow noopener">ghcr.io</a>).<br>
It is impossible to manage the package (set visibility level, delete package).</p>
<p>Reproduction steps:</p>
<ol>
<li>Push the image with dot in the name:</li>
</ol>
<pre><code class="lang-auto">$ docker push ghcr.io/plutzilla/test.with.dot
The push refers to repository [ghcr.io/plutzilla/test.with.dot]
fb7334cb685d: Layer already exists 
550333325e31: Layer already exists 
22ea89b1a816: Layer already exists 
a4d893caa5c9: Layer already exists 
0338db614b95: Layer already exists 
d0f104dc0a1f: Layer already exists 
latest: digest: sha256:3a18a87d290d90e3f437b22968321b65b21316726a15823339f94428c35819f7 size: 1573
</code></pre>
<p>The 404 error is returned upon accessing the package via UI:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a036178e1c6949c3437bf1ec672c4fff2b125e63.png" data-download-href="/uploads/short-url/mRihMJYXQWqzfTcfYE5FEfRms2T.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1925785/181092261-b0a94058-14e8-4ff4-a2f2-9747954742ec.png" alt="image" data-base62-sha1="mRihMJYXQWqzfTcfYE5FEfRms2T" width="690" height="433" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/a036178e1c6949c3437bf1ec672c4fff2b125e63_2_690x433.png, https://aws1.discourse-cdn.com/github/original/2X/a/a036178e1c6949c3437bf1ec672c4fff2b125e63.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/a/a036178e1c6949c3437bf1ec672c4fff2b125e63.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a036178e1c6949c3437bf1ec672c4fff2b125e63_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">721×453 116 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Workaround: do not use dot in the image name.</p>

**Comment by clarkbw** [comment, marked as answer]
<p>We’ve pushed out a fix for this.  Could you verify its working now?  Thx!</p>

**Comment by plutzilla** [comment]
<p>Big thanks - I can now access package settings via UI :slight_smile:</p>

**Comment by plutzilla** [comment]
<p>Did you rollback the change - I cannot access the aforementioned package(s) again?</p>

**Comment by clarkbw** [comment]
<p>Yes, sorry about that.  The dot fix broke packages with a <code>/</code> in the them so we rolled it back. We’ll have a fix for both coming out early next week.  Rails routes ¯_(ツ)_/¯</p>

**Comment by clarkbw** [comment]
<p>Rolled back the rollback. You’re good to go again!</p>

**Comment by plutzilla** [comment]
<p>Great! Thanks again :slight_smile:</p>

**Comment by sudosauer** [comment]
<p>It appears this issue has resurfaced again. Uploading a package with a dot results in a 404. Removing dot fixes the issue.</p>
<p>Has something changed?</p>

**Comment by shayneburgess** [comment]
<p>This has been fixed now and your packages with dots should be visible</p>

**Comment by mattwelke** [comment]
<blockquote>
<p>The dot fix broke packages with a <code>/</code> in the them so we rolled it back. We’ll have a fix for both coming out early next week. Rails routes ¯_(ツ)_/¯</p>
</blockquote>
<p>I love how in a post describing something with a <code>/</code> character breaking, something with a <code>/</code> character broke.</p>
